pkg/operator/ceph/cluster/mon/config.go

```diff
@@ -0,0 +1,277 @@
+/*
```

Comment:
As a general statement, and I think for a different PR, I would like to move the admin/mon key generation and FSID to live outside of the mon operator. I think that it would go along nicely with the new config package I devised.